### DISCUSSION TITLE
Blender animation to CC4

### DISCUSSION THREAD

**Discussion by Gabri94x** [soupday/cc3_blender_tools#220] [Q&A]
hi everyone, i'm trying to to create an animation in blender, then export it for cc4.
I pretty sure i did not understand the workflow can someone address me to the right way?
i have my character in blender with a simple sequence.
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/c66ebc9e-4005-48be-8136-696790d3d6c2)
i tried the live link but, since it is in experimental, is not working well.
i tried to export using the "export" animation only, from the blender plugin 
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/e7400434-93c8-45aa-ad0e-9c4ac16230ef)
but the result i weird in cc4
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/2009b2a3-7ec6-47b5-bc24-63cec8f02e6a)
i tried to import in different ways

1.  external animation using blender metarig profile
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/984216e0-cd59-476f-b952-f9f163493625)
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/adbadfae-ff7c-4e6e-8bd0-c9f7aeb42383)

2 external animation using current character profile. i get the error
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/b1b1f819-3fa3-4e68-914e-55eda4241374)

3. i tried to export both mesh and animation but it fails
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/9d6132be-25f9-40ee-b217-c5f22e83cc86)
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/f8ab6979-bc0c-4d88-a0f1-4784dc7e773f)
![image](https://github.com/soupday/cc_blender_tools/assets/79253682/d22b9fbc-41cb-437c-ad90-b237b6b36d09)

CC4 v.4.31
Blender Pipeline v2.0.5
Blender CC v2.0.5 

am i missing something?

**Comment by soupday** [comment]
In 2.0.5 I was experimenting with exporting the rigified animation with the meta-rig bone structure, so that CC4 would auto recognize it. I had forgot about that and left it in. I had meant to make that optional.

But it does actually work:

- Make sure you export from Blender with the **Include T-pose** option.
- In CC4 it should identify it as a Blender_meta_rig
- In the motion T-pose: add the fbx again. This will use the t-pose in the fbx to correct the arm rotation.
- Then most importantly: _Uncheck_ **Select Root Bone**.  (The meta-rig doesn't have a root bone)

![rigify-motion-import-2](https://github.com/soupday/cc_blender_tools/assets/79094830/eab74fda-635b-46d0-9910-6a3bca9a989e)

And it should work.

![rigify-motion-import-1](https://github.com/soupday/cc_blender_tools/assets/79094830/ede7fdc1-40b2-461d-9861-a3b5d33de9f5)

**Reply by Gabri94x** [reply]
thanks you! i tried and works perfectly!
